ecommerce/static/js/review.js

```diff
@@ -0,0 +1,49 @@
+$(document).ready(function () {
+// catch the form's submit event
+    $('#commentForm').submit(function (event) {
+        // create an AJAX call
+       console.log($(this).serialize());
+       console.log($(this).attr('method'));
+        $.ajax({
+            data: $(this).serialize(), // get the form data
+            type: $(this).attr('method'), // GET or POST
+            url: $('#commentForm').attr('data-url-name'),
+            // on success
+            success: function (response) {
+                //alert(response);
+                $('.close_modal').click();
+                $('.no-row').remove();
+                $('.review-row').last().append(
+                    '<hr class="mt-4 mb-3"/>' +
+                    '<div class="product-review pb-4">' +
+                        '<div class="d-flex justify-content-between">' +
+                            '<div class="d-flex align-items-center me-4 pe-2">' +
+                                '<div style="padding-right: 20px;">' +
+                                    '<i class="far fa-comment-dots"></i>' +
+                                '</div>' +
+                                '<img class ="rounded-circle" width="50" src="\\media\\profile_pics\\avatar.png" alt="User">' +
+                                '<div class="ps-3" style="padding-left: 20px;">' + 
+                                    '<h6 class="fs-sm mb-0">'+response.new_comment.user+'</h6>' +
+                                    '<span class="fs-ms text-muted">'+ response.new_comment.created +'</span>' +
+                                '</div>' +
+                            '</div>' +
+                            '<div class="p-2"><i class="fas fa-reply"></i></div>' +
+                        '</div>'+
+                        
+                        '<div class="ps-3" style="padding-left: 105px;">' +
+                            '<p class="fs-md mb-2">' +
+                                response.new_comment.content +
+                            '</p>' +
+                        '</div>' +
+                    '</div>'
```

Comment:
Ko nên define nguyên đống html vào trong file js như này sẽ rất rồi e nhé. Tách ra thành 1 file templates *.html để render trong js này, tham khảo cách xử lý này nè
https://stackoverflow.com/questions/47936150/python3-django-how-to-use-jquery-in-render-to-string-template